content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -215,7 +215,40 @@ Nevertheless, you must update the list of admission controllers.
 
 ### Azure
 
-You must use `ACS-Engine` to deploy your cluster.
+You can deploy a Kubernetes cluster to Azure via `ACS-Engine` or `AKS` which fully supports Istio as long as Kubernetes 1.9.6 or later is specified.
+
+AKS is a managed Kubernetes service where the master nodes are paid for and managed by Microsoft, no matter the size or number of agent nodes created.  
+
+ACS-Engine provides extensive customization of the Kubernetes cluster being provisioned.  The downloadable `acs-engine` binary processes a simple api-model JSON file into a large Azure resource manager template which automatically deploys a complete Kubernetes cluster.  Master and agent nodes are mangaged entirely and paid for by the customer.
+
+Instructions for AKS:
+
+You can create an AKS cluster via [the `az` cli](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough) or [the Azure portal](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal).  It is only important Kubernetes 1.9.6 or later is specified.  Not all Kubernetes versions are available in all Azure regions.
+
+For the `az` cli option, complete `az login` authentication OR use cloud shell, then run the following commands below.
+
+1. Determine the `Name` for the desired datacenter location
+
+    {{< text bash >}}
+    $ az account list-locations -o table
+    {{< /text >}}
+
+1. Create the resource group and deploy the AKS cluster
+
+    Replace `myResourceGroup` and `myAKSCluster` with desired names, `myLocation` using the value from the above step, and then execute:
+
+    {{< text bash >}}
+    $ az group create --name myResourceGroup --location mylocation
+    $ az aks create --resource-group myResourceGroup --name myAKSCluster --node-count 3 --generate-ssh-keys
+    {{< /text >}}
+
+1. Get the AKS `kubeconfig` credentials
+
+    {{< text bash >}}
+    $ az aks get-credentials --resource-group myResourceGroup --name myAKSCluster
+    {{< /text >}}
+
+Instructions for ACS-Engine:
 
 1. Follow the instructions to get and install the `acs-engine` binary with
```

Comment:
Simplify the sentence:

Follow the [instructions](https://github.com/Azure/acs-engine/blob/master/docs/acsengine.md#install) to get and install the `acs-engine` binary.